packages/grid/x-data-grid-premium/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-data-grid-premium",
-  "version": "6.20.0",
+  "version": "6.20.1",
```

Comment:
Not having `6.20.1` for `@mui/x-data-grid` is going to be a bit strange...
Even docs rely on it and would keep showing only `6.20.0` as the package version. 🙈 🤷 
<img width="406" alt="Screenshot 2024-06-06 at 12 30 27" src="https://github.com/mui/mui-x/assets/4941090/751f8968-db26-4a1e-883f-1d6ef7152666">

WDYT about also releasing `@mui/x-data-grid` and `@mui/x-data-grid-pro` just to avoid it?
Or are you fine with this minor discrepancy @mui/xgrid? 🤔

Reply:
oh yes ... another good point to consider for the version discussion 🤔

Reply:
I guess the script left it out because there have only been changes to the premium package 🤷🏼

Reply:
Yes, and that makes sense, it's just that not releasing the base package would cause slightly undesirable results. 🙈

Reply:
Yeah, I didn't think about the version in th docs. I'm fine with releasing all the data grid packages 👍
Thanks for spotting this!